apps/src/sites/studio/pages/signup.js

```diff
@@ -161,11 +164,12 @@ window.SignupManager = function (options) {
     $(".email-preference").hide();
 
     // Show correct terms below form.
-    $("#student-terms").fadeIn();
-    $("#teacher-terms").hide();
+    $(".student-options").fadeIn();
+    $(".teacher-options").hide();
 
-    // Implicitly accept terms of service for students.
-    $("#user_terms_of_service_version").prop('checked', true);
+    // Force students to explicitly accept terms of service and data transfer agreement.
+    $("#user_terms_of_service_version").prop('checked', false);
+    $("#user_data_transfer_agreement").prop('checked', false);
```

Comment:
Everyone - even people outside of the EU need to accept the data transfer agreement?

Reply:
Good point.  They won't see it, but we should still clear it in all cases.  I'll update the comment.

Reply:
Actually, since the text doesn't change, maybe we don't even need to uncheck it when toggling.